tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -258,6 +258,41 @@ describe('Service Form Modal', function () {
       cy.visitUrl({url: '/services/overview/%2F/create'});
     });
 
+    it('should fill the entire viewport', function () {
+      var isModalFullScreen = true;
+
+      cy.window()
+        .then(function ($window) {
+          cy.get('.modal-full-screen')
+            .should(function ($element) {
+              if ($element[0].clientHeight !== $window.innerHeight || $element[0].clientWidth !== $window.innerWidth) {
+                isModalFullScreen = false;
+              }
+              expect(isModalFullScreen).to.be.equal(true);
+            });
+        });
+    });
+
+    it('should be horizontally and vertically centered in the modal container', function () {
+      cy.get('.modal-body-wrapper')
+        .should(function ($modalWrapper) {
+          const modalWrapperRect = $modalWrapper[0].getBoundingClientRect();
+          const modalWrapperTop = Math.floor(modalWrapperRect.top + (modalWrapperRect.height / 2));
+          const modalWrapperLeft = Math.floor(modalWrapperRect.left + (modalWrapperRect.width / 2));
+
+          const $modalContent = $modalWrapper.find('.row.panel-grid');
+          const modalContentRect = $modalContent[0].getBoundingClientRect();
+          const modalContentTop = Math.floor(modalContentRect.top + (modalContentRect.height / 2));
+          const modalContentLeft = Math.floor(modalContentRect.left + (modalContentRect.left / 2));
+
+          const topPosDifference = Math.floor(modalWrapperTop - modalContentTop);
+          const leftPosDifference = Math.floor(modalWrapperLeft - modalContentLeft);
+
+          expect(topPosDifference <= modalContentTop).to.be.equal(true);
+          expect(leftPosDifference <= modalContentTop).to.be.equal(true);
```

Comment:
I think a threshold of `5` would be sufficient for this.

I just checked again, and I think maybe we shouldn't check the midpoint of `.row.panel-grid` because, because it has a negative margin. We could account for the negative margin, but that makes the test a little more complicated to understand. Perhaps the element `.create-service-modal-service-picker-options` would be good for the vertical midpoint; I just did this manually and the midpoints of the two elements is the exact same.